docs/core/tools/telemetry.md

```diff
@@ -36,47 +36,91 @@ The `dotnet` command is used to launch both apps and the .NET Core Tools. The `d
 
-## License
+&#8224;This metric is published.
 
-The Microsoft distribution of .NET Core is licensed with the [MICROSOFT .NET LIBRARY EULA](https://aka.ms/dotnet-core-eula). This includes the "DATA" section re-printed below, to enable telemetry.
+The feature doesn't collect personal data, such as usernames or email addresses. It doesn't scan your code and doesn't extract sensitive project-level data, such as name, repo, or author (if you set those in your project file). The data is sent securely to Microsoft servers using [Microsoft Azure Application Insights](https://azure.microsoft.com/services/application-insights/) technology, held under restricted access, and published under strict security controls from secure [Azure Storage](https://azure.microsoft.com/services/storage/) systems.
 
-[.NET NuGet packages](https://www.nuget.org/profiles/dotnetframework) use this same license but do not enable telemetry (see [Scope](#scope) above).
+We want to know how the tools are used and if they're working well, not what you're building with the tools. If you suspect that the telemetry is collecting sensitive data or that we're insecurely or inappropriately handling data, please [file an issue in the dotnet/cli repo issues](https://github.com/dotnet/cli/issues) for investigation.
 
-> 2. DATA. The software may collect information about you and your use of the software, and send that to Microsoft. Microsoft may use this information to improve our products and services. You can learn more about data collection and use in the help documentation and the privacy statement at http://go.microsoft.com/fwlink/?LinkId=528096. Your use of the software operates as your consent to these practices.
+## Published data
 
-## Disclosure
+Published data is available quarterly. The columns of a data file are:
+
+- Timestamp
+- Occurrences&#8224;
+- Command
+- Geography&#8225;
+- OSFamily
+- RuntimeID
+- OSVersion
+- SDKVersion
+
+&#8224;The *Occurrences* column displays the aggregate count of that command's use for that row's metrics that day. &#8225;Typically, the *Geography* column displays the name of a country. In some cases, the continent of Antarctica appears in this column, either due to researchers using .NET Core in Antarctica or incorrect location data.
 
-The .NET Core Tools display the following text when you first run one of the commands (for example, `dotnet restore`). This "first run" experience is how Microsoft notifies you about data collection. This same experience also initially populates your NuGet cache with the libraries in the .NET Core SDK, avoiding requests to NuGet.org (or other NuGet feed) for these libraries.
+### Example
```

Comment:
@nguerrera @livarcocc can you guys help to explain how the test runners are published? I see test and xunit entries on the published data, for example.

/cc @richlander